services/QuillLMS/client/app/constants/comprehension.ts

```diff
@@ -87,6 +93,8 @@ export const readingLevelOptions = [
   },
 ];
 
+export const ruleApiOrder = [RULES_BASED_1, OPINION, PLAGIARISM, AUTO_ML, RULES_BASED_2, GRAMMAR,  SPELLING, RULES_BASED_3];
```

Comment:
I hate storing this as a constant on the front-end since it could change, but I don't have a better idea, so we'll just have to live with it.